tests/testthat/test-epi_slide.R

```diff
@@ -300,34 +300,48 @@ for (p in (param_combinations %>% transpose())) {
       concatenate_list_params(p)
     ),
     {
-      out_sum <- rlang::inject(epi_slide(test_data, ~ sum(.x$value), !!!slide_args)) %>%
-        rename(slide_value_value = slide_value)
-      out_mean <- rlang::inject(epi_slide(test_data, ~ mean(.x$value), !!!slide_args)) %>%
-        rename(slide_value_value = slide_value)
```

Comment:
praise: new names/default behavior/code flow is looking a lot better than "value_value" :)